cmd/lakectl/cmd/doctor.go

```diff
@@ -61,56 +61,82 @@ var errorTemplate = `{{ .Message |red }}
+		}
 		if detailedErr, ok := err.(Detailed); ok {
 			Write(detailedErrorTemplate, detailedErr)
 		} else {
 			Write(errorTemplate, err)
 		}
 
+		if verbose {
+			Write(analyzingMessageTemplate, &UserMessage{"Trying to validate access key format."})
+		}
 		accessKeyID := cfg.Values.Credentials.AccessKeyID
 		if !IsValidAccessKeyID(accessKeyID) {
-			Write(analayzingMessageTemplate, &UserMessage{"access_key_id value looks suspicious: " + accessKeyID})
+			Write(analyzingMessageTemplate, &UserMessage{"access_key_id value looks suspicious: " + accessKeyID})
+		} else if verbose {
+			Write(analyzingMessageTemplate, &UserMessage{"Couldn't find a problem with access key format."})
 		}
 
+		if verbose {
+			Write(analyzingMessageTemplate, &UserMessage{"Trying to validate secret access key format."})
+		}
 		secretAccessKey := cfg.Values.Credentials.SecretAccessKey
 		if !IsValidSecretAccessKey(secretAccessKey) {
-			Write(analayzingMessageTemplate, &UserMessage{"secret_access_key value looks suspicious..."})
+			Write(analyzingMessageTemplate, &UserMessage{"secret_access_key value looks suspicious..."})
+		} else if verbose {
+			Write(analyzingMessageTemplate, &UserMessage{"Couldn't find a problem with secret access key format."})
 		}
 
+		if verbose {
+			Write(analyzingMessageTemplate, &UserMessage{"Trying to validate endpoint URL format."})
+		}
 		serverEndpoint := cfg.Values.Server.EndpointURL
 		if !strings.HasSuffix(serverEndpoint, api.BaseURL) {
-			Write(analayzingMessageTemplate, &UserMessage{"Suspicious URI format for server.endpoint_url: " + serverEndpoint + " doesn't end with: `" + api.BaseURL + "`."})
+			Write(analyzingMessageTemplate, &UserMessage{"Suspicious URI format for server.endpoint_url: " + serverEndpoint})
+		} else if verbose {
+			Write(analyzingMessageTemplate, &UserMessage{"Couldn't find a problem with endpoint URL format."})
 		}
 	},
 }
 
-func ListRepositoriesAndAnalyze(ctx context.Context) error {
+func ListRepositoriesAndAnalyze(ctx context.Context, verboseMode bool) error {
 	configFileName := viper.GetViper().ConfigFileUsed()
 	msgOnErrUnknownConfig := "It looks like you have a problem with your `" + configFileName + "` file."
 	msgOnErrWrongEndpointURI := "It looks like endpoint url is wrong."
 	msgOnErrCredential := "It seems like the `access_key_id` or `secret_access_key` you supplied are wrong."
 
+	if verboseMode {
+		Write(analyzingMessageTemplate, &UserMessage{"Trying to get endpoint URL and parse it as an URL format."})
+	}
 	// getClient might die on url.Parse error, so check it first.
 	serverEndpoint := cfg.Values.Server.EndpointURL
 	_, err := url.Parse(serverEndpoint)
 	if err != nil {
 		return &WrongEndpointURIError{msgOnErrWrongEndpointURI, err.Error()}
 	}
 	client := getClient()
+	if verboseMode {
+		Write(analyzingMessageTemplate, &UserMessage{"Trying to run a simple command using current configuration."})
```

Comment:
```suggestion
		Write(analyzingMessageTemplate, &UserMessage{"Trying to run a sanity command using current configuration."})
```
if also accepted above..